metrics-api/src/main/java/software/amazon/swage/metrics/NullContext.java

```diff
@@ -21,12 +22,18 @@ public static NullContext empty() {
     }
 
     private final TypedMap attributes;
+    private final MetricContext parent;
 
     /**
      * @param attributes the attributes of the measurement context
      */
     public NullContext(TypedMap attributes) {
+        this(attributes, null);
+    }
+
+    NullContext(TypedMap attributes, MetricContext parent) {
```

Comment:
Kind of strange that this would be package-private rather than protected - either NullContext can be extended (in which case this constructor seems like it should be protected), or NullContext is not meant to be extended (in which case the class should be final